packages/core-forger/lib/client.js

```diff
@@ -43,15 +42,15 @@ module.exports = class Client {
    * Sends the WAKEUP signal to the to relay hosts to check if synced and sync if necesarry
    */
   async syncCheck () {
-    await Promise.each(this.hosts, async (host) => {
-      logger.debug(`Sending wake-up check to relay node(s) ${host}`)
-
-      try {
-        await this.__get(`${this.host}/internal/syncCheck`)
-      } catch (e) {
-        //
-      }
-    })
+    await this.__chooseHost()
+
+    logger.debug(`Sending wake-up check to relay node ${this.host}`)
+
+    try {
+      await this.__get(`${this.host}/internal/syncCheck`)
+    } catch (e) {
+      logger.error(e.message)
```

Comment:
Is it worth a prefix message to the error here? E.g. `Could not sync check: ${e.message}`